### PR TITLE
Some documentation for the wrapper classes

### PR DESCRIPTION
**Pull request by ancorgs**
Still to be improved, of course.

### CODE REVIEW

File: src/lib/y2storage/blk_device.rb [hunk 1]
```diff
@@ -33,27 +33,75 @@ class BlkDevice < Device
...
+    # @!method name=
+    #   @see #name
     storage_forward :name=
+
+    # @!method region
+    #   @return [Region]
     storage_forward :region, as: "Region"
+
+    # @!method region=
+    #   @see #region
```
Comment (jreidinger) [inline src/lib/y2storage/blk_device.rb]: NP: to be honest I do not like much this see stuff. Do you consider document it as attribute? http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attribute

File: src/lib/y2storage/btrfs_subvolume.rb [hunk 2]
```diff
@@ -29,20 +29,54 @@ module Y2Storage
...
     storage_forward :default_btrfs_subvolume?
-    storage_forward :default_btrfs_subvolume=
+
+    # @!method create_btrfs_subvolume(path)
+    #   @param path [String] path of the new subvolume
+    #   @return [BtrfsSubvolume]
     storage_forward :create_btrfs_subvolume, as: "BtrfsSubvolume"
 
+    # Sets this subvolume as the default one
+    def set_default_btrfs_subvolume
```
Comment (jreidinger) [inline src/lib/y2storage/btrfs_subvolume.rb]: NP: I feel a bit strange about this method name. What about `assign_as_default_btrfs_subvolume` or `default_btrfs_subvolume!`?

Comment (ancorgs) [inline src/lib/y2storage/btrfs_subvolume.rb]: I also kind of dislike the original name in libstorage. But this should become a direct forward to libstorage once the problems with SWIG are fixed. Thus, I would like to keep the original name if there is not a very strong reason to change it, other than our personal preferences.

Comment (jreidinger) [inline src/lib/y2storage/btrfs_subvolume.rb]: OK

**Pull request merged**